### PR TITLE
Add creatureUser redux

### PR DESCRIPTION
**Pull request by Muzishell**

### CODE REVIEW

**Comment by agaryen** [comment]
Marked as WIP but in the 'In Review' column ?
I'm guessing the WIP tag is outdated.

File: client/src/modules/actions/api.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const URL = 'http://91.121.66.56';
```
Comment (agaryen) [inline client/src/modules/actions/api.js]: Hum, not sure we want to have Patrice IP right here.
First, it's not that great having it displayed on a public repository, but I'm pretty sure we won't even deploy our app on Patrice.
You probably should just replace it with localhost for now.

File: client/src/modules/actions/users/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export {
```
Comment (agaryen) [inline client/src/modules/actions/users/index.js]: What about dropping the indexes.js for actions?
We thought at first it was a good idea, but we agreed it was painful when we had to edit names and stuff.

File: client/src/modules/actions/users/thunks.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import URL_API from '../api';
+
+import {
+  usersCreateRequest,
+  usersCreateFailure,
+  usersCreateSuccess,
+} from './creators';
+
+export const a = () => {}
```
Comment (agaryen) [inline client/src/modules/actions/users/thunks.js]: Hum ?

File: client/src/pages/registerPage/RegisterForm.js [hunk 4]
```diff
@@ -78,7 +79,11 @@ const RegisterFormik = withFormik({
     username: yup.string().min(3, 'Username must be 3 characters or longer').required('Username is required')
   }),
   handleSubmit(values, { resetForm, setErrors, setSubmitting }) {
-    // TODO: Call the redux once it's done
+    values.registerUser({
+      email: values.email,
+      username: values.username,
+      password: values.password,
+    });
     setTimeout(() => {
```
Comment (agaryen) [inline client/src/pages/registerPage/RegisterForm.js]: I think we might remove that now.

File: client/src/modules/actions/api.js (hunk 1 above)
Comment (agaryen) [reply client/src/modules/actions/api.js]: f64b36b

File: client/src/modules/actions/users/thunks.js (hunk 3 above)
Comment (Muzishell) [reply client/src/modules/actions/users/thunks.js]: Hum :3

File: client/src/modules/actions/users/index.js (hunk 2 above)
Comment (Muzishell) [reply client/src/modules/actions/users/index.js]: Ok

**Comment by agaryen** [comment]
Let's merge now, and worry about extensive tests and validations later, when register and login are normalized.

**Pull request merged**